docs/apis/internal/agent.md

```diff
@@ -0,0 +1,116 @@
+# Agent API
+
+This API is for communication between the Seldon Scheduler and the Seldon Agent which runs next to each inference server and manages the loading and unloading of models onto the server as well as acting as a reverse proxy in the data plane for handling requests to the inference server.
+
+## Proto Definition
```

Comment:
protobuf needs to be included from source file

Reply:
All references to code files that use the full files have been replaced with the github gist integration of GitBook. See example here https://docs.seldon.ai/seldon-core-v2/~/changes/8TZszKFdZfFvLTBv6Pe7/apis/scheduler